packages/core/PluginLoader.ts

```diff
@@ -128,7 +128,16 @@ export default class PluginLoader {
   async loadCJSPlugin(
     pluginDefinition: CJSPluginDefinition,
   ): Promise<LoadedPlugin> {
-    const parsedUrl = new URL(pluginDefinition.cjsUrl)
+    let parsedUrl: URL
+    try {
+      parsedUrl = new URL(
+        pluginDefinition.cjsUrl,
+        getGlobalObject().location.href,
+      )
+    } catch (error) {
+      console.error(`Error parsing URL: ${pluginDefinition.cjsUrl}`)
+      throw error
```

Comment:
maybe 
```
throw new Error(`Error parsing URL: ${pluginDefinition.cjsUrl}`)
```

?

Reply:
I thought about that, but wanted to keep the original error in case it had any relevant information. I guess that's not likely, though, so maybe I can log the original error and then throw the custom error since that's what users will see in the UI.